packages/logging/src/angular/angular-logger.js

```diff
@@ -8,17 +9,19 @@ class talendLoggerConfiguration {
 	isInitialized = false;
 
 	init({ serverUrl, getState, processState = state => state }) {
-		if (this.isInitialized) {
-			console.error(
-				'@talend/logging : already initialized. This second initialization may not be what you want.',
-			);
-		} else if (!serverUrl) {
-			console.error(
+		invariant(
+			!this.isInitialized,
```

Comment:
I think you didn't understood what the first param is doing.
It's to said if it's only log or throw ;)